pkg/ccl/backupccl/restore_data_processor.go

```diff
@@ -464,7 +464,12 @@ func (rd *restoreDataProcessor) processRestoreSpanEntry(
 		valueScratch = append(valueScratch[:0], iter.UnsafeValue()...)
 		value := roachpb.Value{RawBytes: valueScratch}
 
-		key.Key, ok, err = kr.RewriteKey(key.Key)
+		key.Key, ok, err = kr.RewriteKey(key.Key, key.Timestamp.WallTime)
+
+		if errors.Is(err, ErrImportingKeyError) {
+			// Elide importing keys
```

Comment:
Lets add more to this comment, the logic is subtle so itll be nice to call out what and why we are eliding keys with this error.